ray-operator/controllers/ray/common/pod.go

```diff
@@ -753,10 +759,17 @@ func generateRayStartCommand(ctx context.Context, nodeType rayv1.RayNodeType, ra
 	if _, ok := rayStartParams["num-gpus"]; !ok {
```

Comment:
Does it make sense to gate the custom accelerator check if `num-gpus` doesn't exist? Maybe we need to put the custom accelerator check outside this if block

Reply:
Again, this is based on the assumption that either `num-gpus` or the custom accelerator type can exit on the node.  So that only check for other resource types if `num-gpus` does not exist.

Reply:
Moved it into another `if` block